crm_phonecall_summary_predefined/__init__.py

```diff
@@ -0,0 +1,4 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
please make all license links in the addons secure with https (OCA maintainers told me)

Reply:
Done